matrix/all-in-one.yaml

```diff
@@ -0,0 +1,575 @@
+component: Splunk Distribution of OpenTelemetry .NET
```

Comment:
@theletterf Do you need something like `requirements` that would e.g. specify the minimal required language version, supported OSes, and other constrains?

Reply:
As general metadata, that'd be very useful.